app/main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
+
```

Comment:
* `skills` variable should be an attribute of the instance not class.
* don't replace skills but add or extend them in init
* use `+=` or `extend()`


read [checklist.md](https://github.com/mate-academy/py-software-engineers/blob/master/checklist.md)!

Reply:
Understood. Fixing it.

Reply:
Should I do the same for Full_Stack class as well?